deploy/README.md

```diff
@@ -197,56 +343,44 @@ kubectl -n sumologic apply -f fluentd-sumologic.yaml
 
-```sh
-brew install kubernetes-helm
-```
-
-Apply `tiller-rbac.yaml` manifest with `kubectl`, and deploy Tiller with a service account:
+Download the Prometheus Operator `prometheus-overrides.yaml` by running
 
-```sh
-kubectl apply -f https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/helm/tiller-rbac.yaml \
-  && helm init --service-account tiller
+```bash
+$ cd /path/to/helm/charts/  
+$ curl - LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/v0.4.0/deploy/helm/prometheus-overrides.yaml
 ```
 
-This manifest binds the default `cluster-admin` ClusterRole in your Kubernetes cluster to the `tiller` service account (which is created when you deploy Tiller in the following step.)
-
-Download the Prometheus Operator `prometheus-overrides.yaml` from GitHub:
-
-```sh
-curl -LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/v0.4.0/deploy/helm/prometheus-overrides.yaml
-```
+In `prometheus-overrides.yaml`, edit to define a unique cluster identifier. The default value of the cluster field in the externalLabels section of prometheus-overrides.yaml is kubernetes. If you will be deploying the metric collection solution on multiple Kubernetes clusters, you will want to use a unique identifier for each. For example, you might use “Dev”, “Prod”, and so on.
 
 Before installing `prometheus-operator`, edit `prometheus-overrides.yaml` to define a unique cluster identifier. The default value of the `cluster` field in the `externalLabels` section of `prometheus-overrides.yaml` is `kubernetes`. If you will be deploying the metric collection solution on multiple Kubernetes clusters, you will want to use a unique identifier for each. For example, you might use “Dev”, “Prod”, and so on.
 
 __NOTE__ It’s fine to change the value of the `cluster` field, but don’t change the field name (key).
 
-__NOTE__ If you plan to install Prometheus in a different namespace than you deployed FluentD to in Step 1, or you have an existing Prometheus you plan to apply our configuration to running in a different namespace,  please update the remote write API configuration to use the full service url. e.g. `http://collection-sumologic.sumologic.svc.cluster.local:9888`.
+__NOTE__ If you plan to install Prometheus in a different namespace than you deployed Fluentd to in Step 1, or you have an existing Prometheus you plan to apply our configuration to running in a different namespace,  please update the remote write API configuration to use the full service URL like, `http://collection-sumologic.sumologic.svc.cluster.local:9888`.
 
 You can also [Filter metrics](#filter-metrics) and [Trim and relabel metrics](#trim-and-relabel-metrics) in `prometheus-overrides.yaml`.
 
 Install `prometheus-operator` using Helm:
```

Comment:
Since the user isn't actually using helm to install, let's change the wording here to something more along the lines of "Install `prometheus-operator` by generating the yaml files using Helm" or something like that